client/src/components/Ticketing/mainpage/SubscriptionPurchasing/SeasonSubscriptions.tsx

```diff
@@ -0,0 +1,86 @@
+import React, {useState} from 'react';
+import {getSeasonImageDefault} from '../../../../utils/imageURLValidation';
+import {ImageCarousal} from './ImageCarousal';
+import {CarousalEvent} from './CarousalEvent';
+import {SubscriptionOption} from './SubscriptionOption';
+import {SeasonWithSubscriptions} from './SubscriptionPurchaseUtils';
+
+interface SeasonSubscriptionsProps {
+  season: SeasonWithSubscriptions;
+  setPopUpProps: any;
+}
+
+export const SeasonSubscriptions = (props: SeasonSubscriptionsProps) => {
+  const {season, setPopUpProps} = props;
+  const [anchors, setAnchors] = useState(
+    Array(season.events.length).fill(null),
+  );
+
+  return (
+    <main
+      className='bg-fixed bg-cover'
+      style={{
+        backgroundImage: `url(${getSeasonImageDefault(
+          season.imageurl,
+        )}),url(${getSeasonImageDefault()})`,
+      }}
+    >
+      <div
+        className='flex flex-col justify-center items-center min-h-[calc(100vh-233px)]
+                   md:min-h-[calc(100vh-142px)] backdrop-blur-sm
```

Comment:
For the `md:` could you make the `min-h-screen` so the footer is hidden unless you scroll down

Reply:
Fixed